webui/react/src/components/kit/useNote.tsx

```diff
@@ -0,0 +1,33 @@
+import React, { useState } from 'react';
+
+import NoteCard, { Props as NoteProps } from 'components/kit/NoteCard';
+import NoteCards, { Props as NotesProps } from 'components/kit/NoteCards';
+import { Note } from 'types';
+
+export const useNote = (): ((props?: NoteProps) => JSX.Element) => {
```

Comment:
Consider renaming this to indicate it's providing state for the designkit demo, otherwise people unfamiliar to the codebase might think this is what you're supposed to use when dealing with notes